hasura/metadata/actions.graphql

```diff
@@ -24,12 +30,13 @@ type Mutation {
 
 input create_circle_input {
   user_name: String!
+  address: String!
   circle_name: String!
   protocol_name: String
   protocol_id: Int
 }
 
-input createUserInput {
+input CreateUserInput {
```

Comment:
this casing seems intentional, but most of the custom inputs are not cased like this, they are snakey. again I don't really care which (i don't have a great basis) but i'm fond of consistency